bin/update-translations.sh

```diff
@@ -35,8 +35,8 @@ npm run i18n:compile
       rm -rf "lang/${lower}"
       cp -r lang/en "lang/${lower}"
   done
-  npm run i18n:crowdin-to-typeshed
   npm run i18n:typeshed-to-crowdin
+  npm run i18n:crowdin-to-typeshed
```

Comment:
This only worked the other way around for me previously because I had the file produced by i18n:typeshed-to-crowdin already in place.